### PR TITLE
configs: add example_full.yaml

### PR DESCRIPTION
**Pull request by honwen**

### CODE REVIEW

**Comment by LEXUGE** [comment]
Would you like to clarify your formatting options on JSON and yaml? Else it might be easy to mess up later if I edit them.

**Comment by honwen** [comment]
> Would you like to clarify your formatting options on JSON and yaml? Else it might be easy to mess up later if I edit them.

with default option of https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*.rs]
```
Comment (LEXUGE) [inline .editorconfig:7]: I don't think these are necessary as `rustfmt` automatically takes care of rust formatting.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+
+[*.rs]
```
Comment (LEXUGE) [inline .editorconfig:14]: And also are these conflicting? (it seems like there are two `[*.rs]` sections)

**Comment by honwen** [comment]
editorconfig only effect `*.{md,yml,yaml,json}` now

**Pull request merged**